qa/rpc-tests/wallet_shieldcoinbase.py

```diff
@@ -185,3 +192,19 @@ def verify_locking(first, second, limit):
         sync_mempools(self.nodes[:2])
         self.nodes[1].generate(1)
         self.sync_all()
+
+        print("activate NU5")
+        current_height = self.nodes[0].getinfo()['blocks']
+        self.nodes[0].generate(1300-current_height)
+
+        print("create a unified address")
+        account = self.nodes[0].z_getnewaccount()['account']
+        ua = self.nodes[0].z_getaddressforaccount(account)['unifiedaddress']
+
+        print("shield coinbase to the unified address")
+        result = self.nodes[0].z_shieldcoinbase("*", ua)
+        assert_equal(result["remainingUTXOs"], 50)
```

Comment:
Check what the amount is before the `z_shieldcoinbase()` call, to ensure we are actually having an effect. Additionally, we should have a UA-specific check that ensures `z_getbalanceforaccount` shows that the expected pool is incremented (Sapling before NU5, Orchard after NU5).